environments/drupal-8.sh

```diff
@@ -15,6 +15,24 @@ function drupal_ti_clear_caches() {
 	drush cr
 }
 
+#
+# Ensures that the module is linked into the Drupal code base.
+#
+function drupal_ti_ensure_module_linked() {
+	# Ensure we are in the right directory.
+	cd "$DRUPAL_TI_DRUPAL_DIR"
+
+	# This function is re-entrant.
+	if [ -L "$DRUPAL_TI_MODULES_PATH/$DRUPAL_TI_MODULE_NAME" ]
+	then
+		return
+	fi
+
+	composer config repositories.$DRUPAL_TI_MODULE_NAME path $TRAVIS_BUILD_DIR
+	composer require drupal/$DRUPAL_TI_MODULE_NAME
+	composer update -n --lock
```

Comment:
Looks really nice, can we somehow avoid composer downloading the whole repositories for those step or is that what -n means?

Asking for my friend the drupal_ti cache system :p.

Thanks!

PS: I like the path setup there, quick and easy.